### PR TITLE
MD-listomomentum, initial commit

### PR DESCRIPTION
**Pull request by joecefnhedog**
checkit

### CODE REVIEW

**Comment by joecefnhedog** [comment]
not bad aye

**Comment by joecefnhedog** [comment]
nada

File: src/linkedListo.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+class QueueUsingLL {
+    constructor() {
+        this.length = 0;
+        this.head = null;
+    }
+
+    size() {
+        this.length++
+        this.length--
```
[the remark this answers is not in the document]
Comment (joecefnhedog) [reply src/linkedListo.js:16]: we like classes?

[the remark this answers is not in the document]
Comment (joecefnhedog) [reply src/linkedListo.js:16]: ++ --, was checking you red it

File: src/linkedListo.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            let value = current.element;
+            current = current.next;
+            this.head = current;
+            this.length--;
+            return value;
+        }
+
+        return null;
+    }
+}
```
Comment (Thrillpool) [inline src/linkedListo.js:50]: pls implement [Symbol.iterator], any iterable collection shud have it as it's so useful.

Comment (joecefnhedog) [reply src/linkedListo.js:50]: dont know what that is, but I will say yes boss

Comment (joecefnhedog) [reply src/linkedListo.js:50]: i'll pull it out the bag

File: src/linkedListo.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        var node = new Nova(cane),
+            current;
+
+        if (this.head === null) {
+            this.head = node;
+        } else {
+            current = this.head;
+            while (current.next) {
+                current = current.next;
+            }
```
[the remark this answers is not in the document]
Comment (joecefnhedog) [reply src/linkedListo.js]: no more

File: src/linkedListo.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    size() {
+        this.length++
+        this.length--
+        return this.length;
+    }
+
+    enqueue(cane) {
+        var node = new Nova(cane),
```
[the remark this answers is not in the document]
Comment (joecefnhedog) [reply src/linkedListo.js]: don't know what youre talking about

File: src/linkedListo.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (joecefnhedog) [reply src/linkedListo.js]: donezo

**Pull request merged**